### ISSUE TITLE
When downloading excel sheet from Nextcloud, it results as an Empty document 

### ISSUE THREAD

**Issue by Anthael** [ONLYOFFICE/onlyoffice-nextcloud#296]
**Versions :**
Nextcloud 18.0
OnlyOffice 4.2.0 ( and previous )

**Steps to reproduce:**
If you create, modify, save, directly from only office everything is fine.
But if the same document is downloaded directly from NC, the document is empty ( 0 kb )

**Temporary solution and linked issue**
I spot, if the option autosave ( from onlyoffice document settings is unchecked ), the document is correct.

**Onlyoffice parameter**
Parameter : editorConfig.customization.autosave

Maybe be possible to force the option from the settings to all documents ?

**Comment by thmo** [comment]
There seems to be problem with the synchronization between the the document server and NC. This manifests in multiple tickets at various places.

For some people (me included) it happens regardless of whether the document server is external, or the NC community server app is used, so I would assume there's a problem with the integration app.

See e.g. #292.

**Comment by richardrey** [comment]
Hello, I have the problem too, I have tried both methods from the integrated one, such as using the Document server in a separate container.

**Comment by kuchengrab** [comment]
two weeks worth of work in two associations that I am part of are gone, thanks to this bug. we only noticed now, that our nextcloud has a bunch of empty files ... 😫 
thankfully our nextcloud admin was able to fix it. but this is quite unacceptable ...
Sorry, I know this doesn't help to fix the issue.

**Comment by kuchengrab** [comment]
here is how our admin fixed it:
> there was a stupid rule added to nextcloud 19.0 (without any announcement) that certain URLs (in this case http://127.0.0.1.. localhost) can't be reached anymore - just had to change one URL

so it's connected to the update to nextcloud 19.0 (in our case at least)

**Comment by mtippmann** [comment]
looks like it's a combination of multiple issues - for one the issue mentioned by @kuchengrab that nc19 forbids setting internal url to anything `localhost` or `127.0.0.1` - but there seems to be another issue if internal url is *not* left empty but set to the same value as external url - in this case silently nothing gets saved. Removing the values for any internal URLs fixes the issue here (NC 19, OO-Nextcloud 4.3)

**Comment by Anthael** [comment]
Temporary you can use the autosave option from document advanced setting.
But this option is available per document. No way to set up for all

**Comment by despens** [comment]
Maybe this is related, too? #312

**Comment by LinneyS** [comment]
Hi  @Anthael 
When creating a file immediately with the size (.docx ~7kb). Downloading with size 0 is very strange.
Which version of ONLYOFFICE Document Server are you using?

**Comment by FutureTecSystems** [comment]
Maybe related to this? https://github.com/ONLYOFFICE/onlyoffice-nextcloud/issues/363

**Comment by w202mg** [comment]
Same problem here. Once making any changes they appear in the downloaded version only after ~24h.

**Comment by w202mg** [comment]
Same problem here. Once making any changes they appear in the downloaded version only after ~24h.
